### PR TITLE
Add onboarding support

### PR DESCRIPTION
**Pull request by djabarovgeorge**
Created onboarding support.

added env setup with .env generator (with predefined subscriber id).

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,3 @@
+NEXT_PUBLIC_SUBSCRIBER_ID=on-boarding-subscriber-id-123
```
Comment (djabarovgeorge) [inline .env.example:1]: should we remove this one and add it to the script?

File: package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "setup:project": "node src/scripts/setup-env.files.js",
```
Comment (djabarovgeorge) [inline package.json:13]: not sure about the script name here.

File: src/scripts/setup-env.files.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+const readline = require('readline');
+
+// Copy contents from .env.example to .env
+fs.createReadStream('.env.example').pipe(fs.createWriteStream('.env'));
+
+// Ask for user input
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
```
Comment (djabarovgeorge) [inline src/scripts/setup-env.files.js:11]: here we copy .env.example and create .env with the same content

File: src/scripts/setup-env.files.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  const envFile = fs.readFileSync('.env', 'utf8');
+
+  const updatedEnvFile = envFile.replace(
+    'NEXT_PUBLIC_NOVU_APP_ID=',
+    `NEXT_PUBLIC_NOVU_APP_ID=${appId}`
+  );
+
+  // Write the updated contents back to the .env file
+  fs.writeFileSync('.env', updatedEnvFile);
+  rl.close();
```
Comment (djabarovgeorge) [inline src/scripts/setup-env.files.js:24]: here we override the `NEXT_PUBLIC_NOVU_APP_ID=` with the CLI response

File: package.json (hunk 2 above)
Comment (djabarovgeorge) [reply package.json:13]: maybe we need to change this script name + the script file name because they are coupled to the onboarding flow.

**Review by scopsy — APPROVED** [review]

**Pull request merged**